tests/SelfTest/UsageTests/Misc.tests.cpp

```diff
@@ -18,9 +18,6 @@
 #include <limits>
 #include <array>
 #include <tuple>
-#if defined(CATCH_CONFIG_WINDOWS_SEH)
-#include <windows.h>
```

Comment:
Use `<catch2/internal/catch_windows_h_proxy.hpp>`, it defines the `NO_MINMAX`, `LEAN_AND_MEAN` macros etc.